notifier/postgres/store_test.go

```diff
@@ -20,17 +20,23 @@ func TestingStore(ctx context.Context, t testing.TB) *Store {
 	cfg := db.Config()
 	// This looks backwards, but means that failures get lots of output and
 	// verbose output gets a moderate amount of output.
-	cfg.ConnConfig.LogLevel = pgx.LogLevelInfo
+	tracer := &tracelog.TraceLog{
+		LogLevel: tracelog.LogLevelInfo,
+		Logger:   testingadapter.NewLogger(t),
+	}
 	if testing.Verbose() {
-		cfg.ConnConfig.LogLevel = pgx.LogLevelError
+		tracer = &tracelog.TraceLog{
```

Comment:
nit: can just update the log level here, only, but since it's a test, I don't care that much 😄